index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project Module 1</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <header>
+      <nav id="mobile">
+        <div class="menue-button">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+      </nav>
+      <nav id="web">
+        <a href="index.html"><img id="logo" src= './images/logo.jpg' alt="logo"></a>
+        <nav id="navigator">
+          <a href="./about.html">About</a>
+          <a href="#program">Programs</a>
+          <a>Join</a>
+          <a href="#top">Sponser</a>
+          <a>News</a>
+        </nav>
+      </nav>
+    </header>
+
+    <main>
+      <section id="intro">
+        <img id="intro-background" src="./images/main-bg.png" alt="background">
+        <section id="headline">
+          <h2 id="welcome">Welcome to theater!</h2>
+          <h1>My beloved, who is she?<br>حبيبتي من تكون</h1>
+        </section>
+        <p id="bio">We always wondered who the Nightingale's lover is! Will we find the answer through this brilliant panoramic show, which revolves around the biography of Abdel Halim Hafez?</p>
+        <h3 id="date">Tuesday 17/May - Wednesday 25/May •2022•</h3>
+      </section>
+
+      <section id="program">
+        <h2 class="program-header">Main Program</h2>
+        <section id="acts-container">
+          <section class="act">
+            <img class="act-logo" src="./images/Screen-Shot-1.png" alt="act logo">
+            <h3 class="act-name">Act<br>One</h3>
+            <p class="desciption">An opening with a dance medley of Abdel Halim Hafez's songs.</p>
+          </section>
+          <section class="act">
+            <img class="act-logo" src="./images/Screen-Shot-2.png" alt="act logo">
+            <h3 class="act-name">Act<br>Two</h3>
+            <p class="desciption">An opening with a dance medley of Abdel Halim Hafez's songs.</p>
+          </section>
+          <section class="act">
+            <img class="act-logo" src="./images/Screen-Shot-3.png" alt="act logo">
+            <h3 class="act-name">Act<br>Three</h3>
+            <p class="desciption">An opening with a dance medley of Abdel Halim Hafez's songs.</p>
+          </section>
+          <section class="act">
+            <img class="act-logo" src="./images/Screen-Shot-4.png" alt="act logo">
+            <h3 class="act-name">Act<br>Four</h3>
+            <p class="desciption">An opening with a dance medley of Abdel Halim Hafez's songs.</p>
+          </section>
+          <section class="act">
+            <img class="act-logo" src="./images/Screen-Shot-1.png" alt="act logo">
+            <h3 class="act-name">Act<br>Five</h3>
+            <p class="desciption">An opening with a dance medley of Abdel Halim Hafez's songs.</p>
+          </section>
+        </section>
```

Comment:
- [ ] Kindly use a div tag instead of section tags for each element with the class "act" this will make your project more semantic and easily readable by browsers👍🏿 

- [ ] Please add your see whole program button on the desktop 👍🏿 
- [ ] Please add your own background image for this section

 



Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_219](https://user-images.githubusercontent.com/69638013/170558719-eb71fb05-3ed6-4d85-ab30-40daec857f4a.png) |  ![Screenshot_218](https://user-images.githubusercontent.com/69638013/170558694-78a68b75-aa90-4083-ad34-951d5e7319c2.png)

- Great work so far 🎊 👏🏿

Reply:
Thanks a lot for your suggestions. I did miss these points.